### PR TITLE
Clean up the cluster scripts

### PR DESCRIPTION
**Pull request by Kashomon**
- Add a K8S Version option (and use 1.9.2)
- Switch to Beta track with 1.9.2 EAP
- Add a Zone option
- Explicitly set PROJECT/ZONE
- Make the destroy script less scary
- Make a cluster-up-gpu-small for test-size clusters

### CODE REVIEW

File: cluster/cluster-up-gpu [hunk 1]
```diff
@@ -4,20 +4,26 @@ source ./common.sh
...
-  --enable-kubernetes-alpha \
+gcloud beta container clusters create \
   --num-nodes 5 \
   --accelerator type=nvidia-tesla-k80,count=1 \
   --machine-type n1-standard-2 \
   --disk-size 20 \
   --preemptible \
-  --cluster-ipv4-cidr=10.0.0.0/10 \
-  --zone=asia-east1-a \
+  --cluster-ipv4-cidr=10.0.0.0/10 \ # Allocate a big CIDR block for a big cluster
```
Comment (amj) [inline cluster/cluster-up-gpu]: this is only needed for clusters with > 1008 nodes; maybe make the comment specific to that?

File: cluster/cluster-up-gpu-small [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```
Comment (amj) [inline cluster/cluster-up-gpu-small]: what's the logic on small/large?  seems like we should clarify just how we expect folks to use the repo.  Is this an example code?  is this code that all the people who run clusters check in?

it smacks of the eternal configuration vs code in VCS debates, i guess.  

What do we get from having a second cluster-up script?

File: cluster/deploy-gpu-player.sh [hunk 3]
```diff
@@ -16,4 +16,6 @@
 
 source ./common.sh
 
+# TODO(kashomon): envsubst doesn't exist for OSX. needs to be brew-installed
+# via gettext. Should probably warn the user about that.
```
Comment (amj) [inline cluster/deploy-gpu-player.sh]: in the README?  Otherwise this comment is pretty much all that you need :)

Comment (Kashomon) [reply cluster/deploy-gpu-player.sh]: Added it directly in the command

File: cluster/cluster-up-gpu-small (hunk 2 above)
Comment (Kashomon) [reply cluster/cluster-up-gpu-small]: I've simplified this from large cluster. In particular, the GPUS are non-preemptible.

> it smacks of the eternal configuration vs code in VCS debates, i guess.

I don't have the context here.

File: cluster/cluster-up [hunk 4]
```diff
@@ -4,18 +4,22 @@ source ./common.sh
...
+echo "Using Project:      ${PROJECT}"
+echo "Using Zone:         ${ZONE}"
+echo "Using Cluster Name: ${CLUSTER_NAME}"
+echo "Using K8S Version:  ${K8S_VERSION}"
+
 # Create a Kubernetes cluster
 gcloud beta container clusters create \
   --num-nodes 4 \
   --machine-type n1-standard-8 \
-  --min-cpu-platform "Intel Broadwell" \
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply cluster/cluster-up]: Ack. I'll add it back in

**Comment by Kashomon** [comment]
Addresses https://github.com/tensorflow/minigo/issues/1
- Updates cluster to 1.9.2

**Pull request merged**